packages/contact-center/station-login/tests/station-login/station-login.presentational.tsx

```diff
@@ -9,13 +9,19 @@ describe('StationLoginPresentational', () => {
   it('renders the component name', () => {
     const props = {
       name: 'StationLogin',
-      loginState: 'idle',
-      setLoginState: jest.fn(),
-      ccSdk: {},
-      isAvailable: jest.fn(),
+      login: jest.fn(),
+      logout: jest.fn(),
+      loginSuccess: undefined,
+      loginFailure: undefined,
+      logoutSuccess: undefined,
+      teams: ['team123'],
+      loginOptions: ['EXTENSION', 'AGENT_DN', 'BROWSER'],
+      setDeviceType: jest.fn(),
+      setDialNumber: jest.fn(),
+      setTeam: jest.fn()
     };
     render(<StationLoginPresentational {...props} />);
     const heading = screen.getByTestId('station-login-heading');
     expect(heading).toHaveTextContent('StationLogin');
```

Comment:
Check number of times rendered.

Reply:
This code is temporary so I have not added proper testcases for the presentational component. Do you suggest we handle it differently?

Reply:
I think we should have a check for it right now so this acts as a template @Kesari3008